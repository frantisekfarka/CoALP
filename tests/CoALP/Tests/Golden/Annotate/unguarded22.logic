q(s(X),Y) :- p(X,Y).
p(t(X),Y) :- q(Y,Y).
