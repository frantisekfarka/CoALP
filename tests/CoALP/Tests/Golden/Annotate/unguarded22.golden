q(s(V_2), V_3, transform-func-1(V_-6)) :- p(V_2, V_3, V_-6).
p(t(V_4), V_5, transform-func-2(V_-7)) :- q(V_5, V_5, V_-7).
